lib/list_versions_test.go

```diff
@@ -2,23 +2,23 @@ package lib_test
 
 import (
 	"log"
-	"reflect"
 	"testing"
 
 	"github.com/warrensbox/terraform-switcher/lib"
 )
 
 const (
-	hashiURL = "https://releases.hashicorp.com/terraform/"
+	hashiURL = "https://api.releases.hashicorp.com/v1/releases/terraform"
 )
 
 // TestGetTFList : Get list from hashicorp
+
+/*
```

Comment:
Out of interest: why need to keep this commented out function?

Reply:
We don't :)
That was only there while it was still WIP

Reply:
Should be fixed now